examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -52,13 +79,75 @@
 # ---------------------------------------------------------
 #
 # Hemispheric sagittal cuts
-plotting.plot_glass_brain(localizer_tmap_filename,
+plotting.plot_glass_brain(tmap_filename,
                           title='plot_glass_brain with display_mode="lzr"',
                           black_bg=True, display_mode='lzr', threshold=3)
 
 ###############################################################################
-plotting.plot_glass_brain(localizer_tmap_filename, threshold=0, colorbar=True,
+plotting.plot_glass_brain(tmap_filename, threshold=0, colorbar=True,
                           title='plot_glass_brain with display_mode="lyrz"',
                           plot_abs=False, display_mode='lyrz')
 
+###############################################################################
+# Demo glass brain plotting with contours
+# ----------------------------------------
+# To plot maps with contours, we call the plotting function into variable from
+# which we can use specific display features which are inherited automatically.
+# In this case, we focus on using add_contours
+# First, we initialize the plotting function into "display" and first
+# argument set to None since statistical results must be initialized
+# calling with add_contours
```

Comment:
I would say here "since we want an empty glass brain to plotting the statistical maps with "add_contours"".